modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -6,7 +6,12 @@ endif::[]
 
 image::prepare-to-install-breadcrumb.png[]
 
-Before you download and install Anypoint Service Mesh, you must meet the  software, licensing, and permissions and roles requirements describes in this section.
+Before you download and install Anypoint Service Mesh, make sure that you review and fulfill the software, licensing, and permissions and roles requirements describes in this section. 
+
+== About the Audience
```

Comment:
Consider "Audience Requirements," for parallelism and also specificity.

Reply:
Done! Thanks!